perun-core/src/main/java/cz/metacentrum/perun/core/api/AuditMessagesManager.java

```diff
@@ -106,32 +101,32 @@
 	 void createAuditerConsumer(PerunSession perunSession, String consumerName) throws InternalErrorException, PrivilegeException;
 
 	/**
-	 * Log auditer message
+	 * Log auditer message.
 	 *
-	 * @param sess
-	 * @param message
+	 * @param perunSession perun session
+	 * @param message message to be logged
 	 * @throws InternalErrorException
 	 * @throws PrivilegeException
 	 */
-	void log(PerunSession sess, String message) throws InternalErrorException, PrivilegeException;
+	void log(PerunSession perunSession, String message) throws InternalErrorException, PrivilegeException;
```

Comment:
You don't have to rename sess to perunSession, since we use it pretty widely in the code a everybody knows, what it is (can read description).